### DISCUSSION TITLE
Trying to deploy contract on Linea using Truffle

### DISCUSSION THREAD

**Discussion by ahemtt** [ConsenSys-archive/truffle#5998] [Installation]
Hi, i tried to deploy a contract using Truffle but in the last step, i am getting this error: 
CONNECTION ERROR: Couldn't connect to node http://localhost:24012/rpc.
Truffle v5.8.2 (core: 5.8.2)
Node v16.20.0
I followed the docs: https://docs.linea.build/developers/quickstart/deploy-smart-contract/truffle

Any solution about that?

**Comment by cipaterson** [comment]
Hi @ahemtt,
how are you going on this?  Couldn't connect suggests that something went wrong with the step:
truffle dashboard
which should have opened a browser tab on:
localhost:24012
if not, you should be able to browse there manually and see a running dashboard.

Let us know if you are all good now?

**Reply by adam-lokicode** [reply]
@ahemtt  My team and I are creating a solution heavily inspired by Truffle Dashboards and we just got a partnership with Linea, not sure if you are still looking to make contract deployments easier but feel free to sign up here: [https://www.lokicode.io/](https://www.lokicode.io/).
